Lib/test/_test_multiprocessing.py

```diff
@@ -4084,10 +4088,12 @@ def test_shared_memory_ShareableList_basics(self):
             self.assertEqual(sl.count(b'adios'), 0)
 
         # Exercise creating a duplicate.
-        sl_copy = shared_memory.ShareableList(sl, name='test03_duplicate')
+        pid = os.getpid()
```

Comment:
Is it used?

Reply:
Good catch! I'll remove it after finishing tests on buildbots.

Reply:
```suggestion
```

Reply:
> Good catch! I'll remove it after finishing tests on buildbots.

Sure. Let the buildbots complete their jobs.